crates/bevy_ecs/macros/src/lib.rs

```diff
@@ -265,12 +265,12 @@ pub fn impl_query_set(_input: TokenStream) -> TokenStream {
                 unsafe fn get_param(
                     state: &'s mut Self,
                     system_meta: &SystemMeta,
-                    world: &'w World,
+                    world: &SemiSafeCell<'w, World>,
```

Comment:
`&&T` is kind of weird can we not just take `SemiSafeCell<'w, World>`

Reply:
done